README.md

```diff
@@ -86,6 +86,7 @@ Table of Contents
     * Cloud Build - 120 build-minutes per day
     * Cloud Source Repositories - Up to 5 Users, 50 GB Storage, 50 GB Egress
     * Full, detailed list - https://cloud.google.com/free
+    * [Google Collab](https://colab.research.google.com/) - Free Jupyter Notebooks development environment. 
```

Comment:
I think it would make more sense above the link to their list of free things - the list is logically linked to after all the things this mentions.

Reply:
ya, sorry